### DISCUSSION TITLE
Core.cpp edit for Raspberry Pi on clone of master today

### DISCUSSION THREAD

**Discussion by pmaine** [AlexandreRouma/SDRPlusPlus#306] [Q&A] [answered]
If I am on the git tag for 1.0.3 the edit for a Raspberry pi is on line 275. Where should I make the edit in the newest file?

**Comment by AlexandreRouma** [comment, marked as answer]
Hello,

Modifying the source code is no, longer necessary to run it on the pi, everything is now automatic.

Simply build and install it like on any linux platform

**Reply by pmaine** [reply]
Github wont let me mark this as answered...
